persistence/sql/outbox_modes_sqlpersistence_[6,).partial.md

```diff
@@ -33,20 +54,27 @@ In cases where the outbox transaction spans multiple databases, the `Transaction
 snippet: SqlPersistenceOutboxTransactionScopeMode
 
 In this mode the SQL persistence creates a `TransactionScope` that wraps the whole message processing attempt and within that scope it opens a connection, that is used for:
- - storing the outbox record
- - persisting the application state change applied via `SynchronizedStorageSession`
 
-In addition to that connection managed by NServiceBus, users can open their own database connections in the message handlers. If the underlying database technology supports distributed transactions managed by the Microsoft Distributed Transaction Coordinator (MSDTC) (e.g. SQL Server, Oracle or PostgreSQL), the transaction is escalated to a distributed transaction.
+- storing the outbox record
+- persisting the application state change applied via `SynchronizedStorageSession`
+
+In addition to that connection managed by NServiceBus, users can open their own database connections in the message handlers. If the underlying database technology supports distributed transactions managed by the Microsoft Distributed Transaction Coordinator (MSDTC) (e.g. SQL Server or Oracle), the transaction is escalated to a distributed transaction.
 
 The `TransactionScope` mode is most useful in legacy scenarios such as when migrating from the MSMQ transport to a messaging infrastructure that does not support MSDTC. In this scenario, it is no longer possible to use a distributed transaction which includes the transport and the database. To maintain consistency, the outbox must be used instead. If the outbox table cannot be added to the legacy database, it may be placed in a separate database, but access to both databases must be included in distributed transactions.
 
-#### Transaction Isolation Level
+#### Isolation Level
 
-If required, the outbox transaction isolation level may be adjusted:
+If required, the transaction [isolation level](https://learn.microsoft.com/en-us/dotnet/api/system.transactions.isolationlevel?) may be adjusted:
 
-snippet: SqlPersistenceOutboxIsolationLevel
+snippet: SqlPersistenceOutboxTransactionScopeMode
```

Comment:
If the snippet name changed why doesn't it appear in the snippet code as a change?

Reply:
It already existed